### ISSUE TITLE
Question - History fit_generator() gives None

### ISSUE THREAD

**Issue by Rodriguevb** [keras-team/keras#9759]
Hello, i don't understand how to get `accuracy` at the end of `fit_generator()`

	history = model.fit_generator( someparameters )
	print( history.on_train_end() ) # gives None

same for

	class My_Callback(keras.callbacks.Callback):
		def on_train_begin(self, logs={}):
			self.l = {}
		def on_train_end(self, logs={}):
			self.l = copy.deepcopy(logs)
			return self.l
		def __repr__(self):
			return str(self.l)
	history = My_Callback()
	model.fit_generator( someparameters,  callbacks=[history])
	print( history ) # gives None

I want the `acc` value at the end of `fit_generator`

	25/25 [==============================] - 17s 663ms/step - loss: 2.7852 - acc: 0.1358

thank u
